tasks/install-app.yml

```diff
@@ -69,8 +69,6 @@
   bundler:
     state: present
     chdir: "{{ rails_app_install_path }}"
-    deployment_mode: true
-    exclude_groups: "{{ rails_app_deployment_exclude_groups }}"
```

Comment:
@nomadicoder can you please post what exactly is the deprecation warning you are seeing for this command.  I'm not seeing any warnings in the documentation for the bundler command that this flag is going away: 

https://docs.ansible.com/ansible/latest/collections/community/general/bundler_module.html#parameter-deployment_mode

Is it possible that the warning is more about our use of true/false here vs. yes/no?  We need the deployment flag to remain as is so that gems get installed in the vendor folder of the project group.

Why are you also getting rid of the exclude_groups flag?  That will affect all the playbooks where we currently set an exclude_groups.

Reply:
https://app.circleci.com/pipelines/github/tulibraries/gencon50/114/workflows/69b389a6-1349-449c-aa0f-553540faf877/jobs/185?invite=true#step-103-637

Reply:
@nomadicoder please add that exact message to the description of this PR.  Explain that it is bundler itself that is failing and not an ansible issue.

Explain that my removing these we will either need to handle these features by adding a bundler config to the repos that need this or that we will need to add a default bundler config via this role.

Investigate whether or not it is possible to override these configurations via environmental variables in bundler.  If so move these configurations to environment variables that can be set via this role and overridden by the playbooks.

Investigate whether it's possible to disregard a warning/error in the task and try that as an option instead of just removing this without having a plan for how we can continue to have this functionality.

Investigate whether it's possible for you to pin the version of bundler that geocon is using for now that you do not get this warning or error and add a ticket to devops for us to follow through on this more thoroughly.

Thank you.